projects/storefrontstyles/scss/cxbase/blocks/forms.scss

```diff
@@ -161,6 +161,10 @@ label {
       font-weight: var(--cx-font-weight-semi);
     }
   }
+
+  abbr[title='required'] {
```

Comment:
Should we tie the translation with the selector? Wouldn't changing the language stop the asterisks from being red?